src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserStateObjectFactory.Windows.cs

```diff
@@ -13,13 +13,9 @@ internal sealed class TdsParserStateObjectFactory
 
         public static readonly TdsParserStateObjectFactory Singleton = new TdsParserStateObjectFactory();
 
-
-        // Temporary disabling App Context switching for managed SNI.
         // If the appcontext switch is set then Use Managed SNI based on the value. Otherwise Managed SNI should always be used.
-        //private static bool shouldUseLegacyNetorking;
-        //public static bool UseManagedSNI { get; } = AppContext.TryGetSwitch(UseLegacyNetworkingOnWindows, out shouldUseLegacyNetorking) ? !shouldUseLegacyNetorking : true;
-
-        public static bool UseManagedSNI { get; } = false;
+        private static bool shouldUseLegacyNetorking;
```

Comment:
Actually, is this used anywhere else?  I'm wondering if it's even necessary, and if the subsequent static could instead be:
```C#
public static bool UseManagedSNI { get; } = AppContext.TryGetSwitch(UseLegacyNetworkingOnWindows, out bool shouldUseLegacyNetorking) ? !shouldUseLegacyNetorking : true; 
```